sdk/spring/azure-spring-servicebus/src/main/java/com/azure/spring/servicebus/resourcemanager/ServiceBusQueueProvisioner.java

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-package com.azure.spring.servicebus.core;
+package com.azure.spring.servicebus.resourcemanager;
```

Comment:
How about we name this package as xxx.provisioning, Just like org.springframework.cloud.stream.provisioning.ProvisioningProvider?

Reply:
Good idea!